pkg/leeway/scripts.go

```diff
@@ -253,7 +253,7 @@ func executeBashScript(script string, wd string, env []string) error {
 	err = cmd.Run()
 	if exiterr, ok := err.(*exec.ExitError); ok {
 		if status, ok := exiterr.Sys().(syscall.WaitStatus); ok {
-			os.Exit(status.ExitStatus())
+			return xerrors.Errorf("failed with exit code %d", status.ExitStatus())
```

Comment:
We can't use os.Exit anymore as we might now be executing multiple scripts, so if one fails, we shouldn't exit the entire leeway process.

Reply:
Indeed. This should have never been `os.Exit` in the first place 🤦